specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2020-01-01/SecurityInsights.json

```diff
@@ -664,364 +939,1104 @@
+          "description": "The time the bookmark was created",
+          "format": "date-time",
+          "type": "string"
+        },
+        "createdBy": {
+          "$ref": "#/definitions/UserInfo",
+          "description": "Describes a user that created the bookmark",
+          "type": "object"
+        },
+        "displayName": {
+          "description": "The display name of the bookmark",
+          "type": "string"
+        },
+        "labels": {
+          "description": "List of labels relevant to this bookmark",
+          "items": {
+            "$ref": "#/definitions/Label"
+          },
+          "type": "array"
+        },
+        "notes": {
+          "description": "The notes of the bookmark",
+          "type": "string"
+        },
+        "query": {
+          "description": "The query of the bookmark.",
+          "type": "string"
+        },
+        "queryResult": {
+          "description": "The query result of the bookmark.",
+          "type": "string"
+        },
+        "updated": {
+          "description": "The last time the bookmark was updated",
+          "format": "date-time",
           "type": "string"
+        },
+        "updatedBy": {
+          "$ref": "#/definitions/UserInfo",
+          "description": "Describes a user that updated the bookmark",
+          "type": "object"
+        },
+        "incidentInfo": {
+          "$ref": "#/definitions/IncidentInfo",
+          "description": "Describes an incident that relates to bookmark",
+          "type": "object"
         }
       },
+      "required": [
+        "displayName",
+        "query"
+      ],
       "type": "object"
     },
-    "ActionResponse": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
-      "description": "Action for alert rule.",
+    "ClientInfo": {
+      "description": "Information on the client (user or application) that made some action",
       "properties": {
-        "etag": {
-          "description": "Etag of the action.",
+        "email": {
+          "description": "The email of the client.",
           "type": "string"
         },
-        "properties": {
-          "$ref": "#/definitions/ActionResponseProperties",
-          "description": "Action properties for get request",
-          "x-ms-client-flatten": true
+        "name": {
+          "description": "The name of the client.",
+          "type": "string"
+        },
+        "objectId": {
+          "description": "The object id of the client.",
+          "format": "uuid",
+          "type": "string"
+        },
+        "userPrincipalName": {
+          "description": "The user principal name of the client.",
+          "type": "string"
         }
       },
       "type": "object"
     },
-    "ActionResponseProperties": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/ActionPropertiesBase"
-        }
-      ],
-      "description": "Action property bag.",
+    "CloudError": {
```

Comment:
Please use the common ddefintion, like this: https://github.com/Azure/azure-rest-api-specs/blob/9df7c71f9b955ea66fc9ede43842f937ca53a646/specification/resources/resource-manager/Microsoft.Resources/stable/2019-08-01/resources.json#L4232

Reply:
done

Reply:
Apparently, this is a breaking change - the breaking change check is failing after that change
is it fine @markcowl ?